msteams-platform/toolkit/upgrade-teams-to-work-with-outlook-and office.md

```diff
@@ -0,0 +1,139 @@
+
+The following are the prerequisites to upgrade manifest and `TeamsJS` client SDK:
+
+### Upgrade manifest
+
+Run the command `upgrade Teams manifest` to work in Outlook and Office.com and select Teams Tab app manifest file to migrate:
+
+1. `update manifest` version to use the latest `m365DevPreview` schema.
+1. `update manifest` file to use the latest `DevPreview` schema.
+
+To know more about the required manifest schema and version, see [Developer Preview manifest schema](/microsoftteams/platform/resources/schema/manifest-schema-dev-preview)
+
+### Upgrade Teams JavaScript Client SDK
+
+Run the command `Teams: Upgrade Teams JS SDK to extend in Outlook and Office` and select Teams app project to migrate:
+
+1. Update `TeamsJS SDK` to the latest beta version.
+1. Update function references, `Enum` and interface references.
+1. Add `TODO` comments to finish change callbacks manually.
+
+    > [!TIP]
+    > Changing callbacks alters control flow. This code will have `TODO` comments for your review. 
+    > The typical solution is to make parent function `async` and use `await`.
+
+1. Add `TODO` in places where you get changes to the context schema.
+
+## Run your Teams application in Outlook and Office.com
+
+After upgrading you can run Teams application in Outlook and Office.com.
+
+### Upload and run your application in Teams
+
+The following are the steps to Test your application in Outlook and Office.com by uploading your application through Teams client:
+
+1. Push code changes to the server to host your application.
+1. Move application to a zip folder.
+1. Go to Teams client and select **Apps**.
+1. Select **Upload a custom app** and select your application's zip folder.
+1. Select **Add** on app details to install the application.
+
+Teams install and launch your app. You can find your app in **More apps**.
+
+ ![More apps](../assets/images/upgrade-teams/more-apps.png)
+
+## Run Teams application in Outlook
+
+Perform the following steps to preview personal tab apps in Outlook web app and desktop clients:
+
+### Outlook web application
+
+1. Go to https://outlook.office.com 
+1. Select the three dots on the bottom left bar.
+
+    ![More apps view](../assets/images/upgrade-teams/apps.png)
+
+1. Select the name of your app to preview in Outlook web application.
+
+    ![App preview in outlook](../assets/images/upgrade-teams/preview-outlook-web-application.png)
+
+### Outlook desktop client
+
+1. Open Outlook desktop client.
+1. Select the three dots on the bottom left bar.
+1. Select the name of your app to preview it in Outlook Desktop Client.
+
+## Run Teams application in Office.com
+
+Perform the following steps to preview your apps in Outlook web client:
+
+1. Go to www.office.com/m365apps
+1. Find apps listed below “M365 Apps”
+
+    ![Apps in M365](../assets/images/upgrade-teams/m365-app.png)
```

Comment:
As discussed and mentioned in the comment above, please change the two images in this section.